components/team_settings_modal/team_settings_modal.tsx

```diff
@@ -69,7 +69,7 @@ export default class TeamSettingsModal extends React.PureComponent<Props, State>
             activeTab: 'general',
```

Comment:
Gnarly, up above on line we 53 we're using jquery to do some stuff. Didn't realize we had some dusty ol' jQuery corners hangin' around in webapp.

Reply:
Yeah, there was a campaign to clear it up last year, but I can't remember if any of the tickets that were picked up even got finished. At this point, it might be worth recreating those tickets to bump them to the front of the list.